ui/oopsyraidsy/data/00-misc/general.ts

```diff
@@ -79,6 +87,35 @@ const shieldEffectIdToAbilityId: { [id: string]: string } = {
   'A53': '5EF7', // panhaimatinon -> panhaima
 };
 
+const targetDamageAbilityIdToDuration: { [id: string]: number } = {
+  '1D0C': 20, // chain stratagem
+  '905D': 20, // dokumori
+  '2C93': 15, // off-guard
+  '2C9D': 15, // peculiar light
+  '8707': 60, // breath of magic
+  '8713': Number.MAX_SAFE_INTEGER, // mortal flame (infinite duration)
```

Comment:
Not something that needs to be solved in this PR by any means, but should there maybe be a 'todo' to add a trigger that will update the damage tracker duration if this status effect is lost?  (I might be mis-remembering, but I think mortal flame gets dropped if the source player dies or otherwise goes out of combat?)

Reply:
Outside of a couple of edge cases, Mortal Flame persists until there is a complete combat end/reset. There's even an edge case with a ShB FATE boss chain where it persists between different FATEs.